samples/bluetooth/mesh/light/README.rst

```diff
@@ -34,34 +38,39 @@ This sample application has the following composition data:
    Gen. OnOff Server
    =================  =================  =================  =================
 
-The :ref:`bt_mesh_onoff_srv_readme` instances in elements 1-4 control LEDs 1-4 respectively.
-The Config server allows configurator devices to configure the node remotely.
-The Health server is used to call attention to the device during provisioning.
+The models are used for the following purposes:
+
+* :ref:`bt_mesh_onoff_srv_readme` instances in elements 1 to 4 each control LEDs 1 to 4, respectively.
+* Config Server allows configurator devices to configure the node remotely.
+* Health Server is used to call attention to the device during provisioning.
```

Comment:
@trond-snekvik , can you provide more details about the role of Health Server? "call atention" is a bit generic...

Reply:
Okay. Something like this?

> Health Server provides ``attention`` callbacks that is used to call attention to the device during provisioning, through blinking the device LEDs.

Reply:
Discussed f2f and edited a bit more.